datasets/few_rel/few_rel.py

```diff
@@ -0,0 +1,189 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""FewRel Dataset."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_CITATION = """@inproceedings{han-etal-2018-fewrel,
+    title = "{F}ew{R}el: A Large-Scale Supervised Few-Shot Relation Classification Dataset with State-of-the-Art Evaluation",
+    author = "Han, Xu and Zhu, Hao and Yu, Pengfei and Wang, Ziyun and Yao, Yuan and Liu, Zhiyuan and Sun, Maosong",
+    booktitle = "Proceedings of the 2018 Conference on Empirical Methods in Natural Language Processing",
+    month = oct # "-" # nov,
+    year = "2018",
+    address = "Brussels, Belgium",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/D18-1514",
+    doi = "10.18653/v1/D18-1514",
+    pages = "4803--4809"
+}
+
+@inproceedings{gao-etal-2019-fewrel,
+    title = "{F}ew{R}el 2.0: Towards More Challenging Few-Shot Relation Classification",
+    author = "Gao, Tianyu and Han, Xu and Zhu, Hao and Liu, Zhiyuan and Li, Peng and Sun, Maosong and Zhou, Jie",
+    booktitle = "Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP)",
+    month = nov,
+    year = "2019",
+    address = "Hong Kong, China",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/D19-1649",
+    doi = "10.18653/v1/D19-1649",
+    pages = "6251--6256"
+}
+"""
+
+_DESCRIPTION = """\
+FewRel is a large-scale few-shot relation extraction dataset, which contains more than one hundred relations and tens of thousands of annotated instances cross different domains.
+"""
+
+_HOMEPAGE = "https://thunlp.github.io/"
+
+_LICENSE = "https://raw.githubusercontent.com/thunlp/FewRel/master/LICENSE"
+
+DATA_URL = "https://raw.githubusercontent.com/thunlp/FewRel/master/data/"
+_URLs = {
+    "train_wiki": DATA_URL + "train_wiki.json",
+    "val_nyt": DATA_URL + "val_nyt.json",
+    "val_pubmed": DATA_URL + "val_pubmed.json",
+    "val_semeval": DATA_URL + "val_semeval.json",
+    "val_wiki": DATA_URL + "val_wiki.json",
+    "pid2name": DATA_URL + "pid2name.json",
+    "pubmed_unsupervised": DATA_URL + "pubmed_unsupervised.json",
+}
+
+
+class FewRelDataset(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class FewRel(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `few_rel`